### ISSUE TITLE
The Licensed was changed?

### ISSUE THREAD

**Issue by qindj** [bopoda/ace#12]
Hello, 

I just notice that the license file was changed, does it means the repo would not be MIT any more?

**Comment by bopoda** [comment]
Yes, it can`t be under MIT. Read plz https://wrapbootstrap.com/theme/ace-responsive-admin-template-WB0B30DGR about license Single application and other.
